.readthedocs.yml

```diff
@@ -47,13 +47,8 @@ build:
     - dvipng
     - texlive-pictures
   jobs:
-    pre_install:
-      - export PACKAGE_NAME=pytorch
-
-# Optionally set the version of Python and requirements required to build your docs
-python:
-  install:
-    - requirements: requirements/pytorch/docs.txt
-    #- requirements: requirements.txt
-    - method: pip
-      path: .
+    post_install:
+      - PACKAGE_NAME=lite pip install -e . -f 'https://download.pytorch.org/whl/cpu/torch_stable.html'
+      - git checkout -- setup.py MANIFEST.in
+      - PACKAGE_NAME=pytorch pip install -e .
+      - pip install -r requirements/pytorch/docs.txt
```

Comment:
With this patch, the build succeeds:
- https://readthedocs.org/projects/pytorch-lightning/builds/18533983/

but if #15493 lands first, the change can be simplified accordingly.